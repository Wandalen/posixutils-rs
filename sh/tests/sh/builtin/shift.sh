set a b c d e

shift 0
echo $?
echo $*

shift 2
echo $?
echo $*

