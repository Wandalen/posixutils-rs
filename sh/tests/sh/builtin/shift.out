0
a b c d e
0
c d e
0
d e
